src/ResourceOperations/ResourceHolder.cs

```diff
@@ -206,8 +208,32 @@ private void UpdateFile(string filename, string valueColumnId, bool skipNontrans
             }
 
             // Write the cached resources to the drive
-            using (var writer = new ResXResourceWriter(filename))
+            using (var writer = new ResXResourceWriter(filename))//$"{filename.Replace(".resx","")}_.resx"))
             {
+                //Retiramos el atributo de solo lectura, si existe
+
+                FileAttributes attrs = File.GetAttributes(filename);
+                if (attrs.HasFlag(FileAttributes.ReadOnly))
+                    File.SetAttributes(filename, attrs & ~FileAttributes.ReadOnly & ~FileAttributes.Hidden);

```

Comment:
Move the attribute change code after the file is finished writing and the file handle is closed, so after the `using` scope ends. Doing this while the file is opened in the writer might fail or work in an unexpected way.

Reply:
I'm with you